README.md

```diff
@@ -389,3 +389,14 @@ select * from root.fit.d1
 
 # Frequent Questions for Compiling
 see [Frequent Questions when Compiling the Source Code](https://iotdb.apache.org/Development/ContributeGuide.html#_Frequent-Questions-when-Compiling-the-Source-Code)
+
+# Contact Us
+### QQ Group
+
+* Apache IoTDB User Group: 659990460
+
+### Wechat Group
+
```

Comment:
Actually, I try to search tietouqiao in Wechat, but it's not @qiaojialin ...

Reply:
It's the backup wechat account of Dr. Qiao 🎺~